good-enough-practices-for-scientific-computing.tex

```diff
@@ -846,28 +846,30 @@ \subsection*{Version Control Systems}
 
 \subsection*{What Not to Put Under Version Control}
 
-Despite the benefits of version control systems, some types of files
-may \emph{not} be appropriate for version control.
+The benefits of version control systems don't apply equally to all file types.
+In particular, version control can be more or less rewarding depending on file
+size and format.
 
 First, today's version control systems are not designed to handle
 megabyte-sized files, never mind gigabytes, so large data or
```

Comment:
I gave the GitHub limit of 100MB as a relevant definition of "uncomfortably large".